openehr-rm/src/main/java/com/nedap/archie/rm/datavalues/quantity/datetime/TimeDefinitions.java

```diff
@@ -0,0 +1,54 @@
+package com.nedap.archie.rm.datavalues.quantity.datetime;
+
+import java.time.temporal.ChronoUnit;
+import java.time.temporal.TemporalAmount;
+import java.time.temporal.TemporalUnit;
+
+public class TimeDefinitions {
+
+    //as openehr defines them, different from what java time uses!
+    public static double NOMINAL_DAYS_IN_YEAR = 365.24;
+    public static double NOMINAL_DAYS_IN_MONTH = 30.42;
```

Comment:
Maybe use AVERAGE_DAYS_IN_YEAR/MONTH here as the definition of [Iso8601_duration.to_seconds](https://specifications.openehr.org/releases/BASE/latest/foundation_types.html#_iso8601_duration_class) says
> [...] the corresponding 'average' durations from Time_definitions are used to compute the result.

The numeric values are the same.